lib/cocoapods/generator/module_map.rb

```diff
@@ -42,8 +42,9 @@ def save_as(path)
       # @return [String]
       #
       def generate
+        header = target.requires_frameworks? ? "framework module" : "module"
```

Comment:
In a documentation they are using a term ```specifier```. What about calling it ```module_specifier```?

http://clang.llvm.org/docs/Modules.html#